### ISSUE TITLE
table grouping with category or tag

### ISSUE THREAD

**Issue by mnlbox** [hasura/graphql-engine#1945]
Hi guys,
This is a console feature request.
If we have ability to group tables together (with ability to add some tag or category to each table) we can show table as a tree in left menu in console.
something like this:
![react-bootstrap-treeview](https://user-images.githubusercontent.com/25859075/51655605-9f5fb480-1fb2-11e9-929f-f8c07901c605.jpg)
For example I have an schema with 46 tables. Some of this table is related to each other for example:

1. person
2. person location
3. person assets
4. store
5. store location
6. store cart

if I can add some tag to table and search it or group with this tag it can very useful.
For example we can add `person` tag to `1, 2, 3` and also `store` tag to `4, 5, 6` and also `location` tag to `2, 5` With these type of table grouping (with tag or hierarchy category we can find this table between my 46 table more simpler)

I think it's a good UX improvement.

**Comment by mnlbox** [comment]
@rikinsk, @dsandip, @marionschleifer, @shahidhk Guys I have near 400 table in one of my project now and really can't use my console app :disappointed_relieved:
I'm lost among the tables.
I really need some feature for find and access to tables easily.

**Comment by mnlbox** [comment]
@marionschleifer Any update/plan on this feature?

**Comment by m4ttheweric** [comment]
@manasag I don't think this is a v3 issue unless I am missing something? The original issue was posted in 2019.
